lang/python/homeassistant/patches/0001-Update-dependencies-to-their-latest-version.patch

```diff
@@ -0,0 +1,81 @@
+From fbaecea5672f5ea687d14f01e654b1c22c1f3c15 Mon Sep 17 00:00:00 2001
+From: Josef Schlehofer <pepe.schlehofer@gmail.com>
+Date: Sun, 7 Jul 2019 08:49:31 +0200
+Subject: [PATCH] Update dependencies to their latest version
+
+The latest version of Home Assistant 0.95.4 doesn't work for me.
+See for more details https://github.com/home-assistant/home-assistant/issues/24982
+
+Home Assistant uses pinned versions of dependencies, let's change them to
+their latest version as they are already included in newer versions.
+I'm thinking as it was suggested by jefferyto a in pull request to
+include it in OpenWrt:
+https://github.com/openwrt/packages/pull/8750#issuecomment-485938034
+to remove entire the pinning version
+---
+ homeassistant/package_constraints.txt | 14 +++++++-------
+ setup.py                              | 14 +++++++-------
+ 2 files changed, 14 insertions(+), 14 deletions(-)
+
+diff --git a/homeassistant/package_constraints.txt b/homeassistant/package_constraints.txt
+index a4a08af12..bf72b2825 100644
+--- a/homeassistant/package_constraints.txt
++++ b/homeassistant/package_constraints.txt
+@@ -2,17 +2,17 @@ aiohttp==3.5.4
+ astral==1.10.1
+ async_timeout==3.0.1
+ attrs==19.1.0
+-bcrypt==3.1.6
+-certifi>=2018.04.16
+-jinja2>=2.10
++bcrypt==3.1.7
++certifi>=2019.06.16
++jinja2>=2.10.1
+ PyJWT==1.7.1
+-cryptography==2.6.1
++cryptography==2.7
+ pip>=8.0.3
+ python-slugify==3.0.2
+ pytz>=2019.01
+-pyyaml>=3.13,<4
+-requests==2.21.0
+-ruamel.yaml==0.15.94
```

Comment:
we may want to find a better mechanism here than patching it like this;
i have no idea for one atm;
but we'll see

Reply:
I wouldn't say that it is so difficult as packages in stable releases (OpenWrt 18.06 and 19.07) are not updated so often.

Right now, there are two solutions

1. Keep tracking changes here and in Hass
2. Remove the entire pinning which was suggested by @jefferyto

Reply:
no strong preference from my side;
a proper mechanism [that works for all/most cases] would be a bit of work anyway [at this point in time];

it's not one of those battles that's worth picking right now;
there are more important things to do until that;